examples/exampleApi.go

```diff
@@ -0,0 +1,66 @@
+package examples
+
+import (
+  "fmt"
+  "net/url"
+  "net/http"
+  "github.com/openwhisk/go-whisk/whisk"
+)
+
+func testCreateAction() {
```

Comment:
This is badly named (by me).  This isn't really a test, just an example of how to use the API.